main.py

```diff
@@ -0,0 +1,140 @@
+import os
+import requests
+import boto3
+import logging
+from dotenv import load_dotenv
+from json_log_formatter import JsonFormatter
+import json
+from datetime import datetime
+import zipfile
+
+def setup_logger():
+    logger = logging.getLogger('TerraformBackupScript')
+
+    if not logger.handlers:
+        handler = logging.StreamHandler()
+        formatter = JsonFormatter()
+        handler.setFormatter(formatter)
+        logger.addHandler(handler)
+
+    logger.setLevel(logging.INFO)
+    return logger
+
+def get_env_vars():
+    env_vars = {
+        'ORGANIZATION': os.getenv('ORGANIZATION'),
+        'S3_BUCKET': os.getenv('S3_BUCKET'),
+        'TOKEN': os.getenv('TOKEN')
+    }
+    
+    missing_vars = [var for var, value in env_vars.items() if not value]
+
+    if missing_vars:
+        for var in missing_vars:
+            logger.error(f'Environment variable {var} is not set.')
+        exit(1)
+
+    return env_vars
+
+def get_workspaces(ORGANIZATION, TOKEN):
+    headers = {
+        'Authorization': f'Bearer {TOKEN}',
+        'Content-Type': 'application/vnd.api+json'
+    }
+
+    url = f'https://app.terraform.io/api/v2/organizations/{ORGANIZATION}/workspaces'
```

Comment:
Can you pull this into a constant/variable? Maybe `TERRAFORM_API_URL` at the very top?